test/e2e/map-modify-actions.test.js

```diff
@@ -0,0 +1,138 @@
+const {assert, getDriver} = require('vl-ui-core').Test.Setup;
+const VlMapModifyActionsPage = require('./pages/vl-map-modify-actions.page');
+
+describe('vl-map-modify-action', async () => {
+  let vlMapPage;
+  const willebroek = [150390.63, 193371.12];
+
+  before(() => {
+    vlMapPage = new VlMapModifyActionsPage(getDriver());
+    return vlMapPage.load();
+  });
+
+  it('als gebruiker kan ik een polygoon aanpassen op een kaart', async () => {
+    const action = await vlMapPage.getModifyPolygonAction();
+
+    const layer = await action.getLayer();
+    await getDriver().wait(async () => (await layer.getNumberOfFeatures()) === 1);
+
+    const polygonFeature = await layer.getFeature(1);
+    const polygonCoordinates = polygonFeature.geometry.coordinates;
+    assert.lengthOf(polygonCoordinates, 1);
+
+    const points = polygonCoordinates[0];
+    assert.lengthOf(points, 4);
+
+    const gent = points[0];
+    assert.lengthOf(gent, 2);
+
+    const mechelen = points[1];
+    assert.lengthOf(mechelen, 2);
+
+    const antwerpen = points[2];
+    assert.lengthOf(antwerpen, 2);
+
+    const leuven = points[3];
+    assert.lengthOf(leuven, 2);
+
+    await action.movePointByCoordinates(gent, willebroek);
+
+    await getDriver().wait(async () => (await layer.getNumberOfFeatures()) === 1);
+    features = await layer.getFeatures();
+    assert.lengthOf(features, 1);
+
+    const modifiedPolygonFeature = await layer.getFeature(1);
+    const modifiedPolygonCoordinates = modifiedPolygonFeature.geometry.coordinates;
+    assert.lengthOf(modifiedPolygonCoordinates, 1);
+
+    const modifiedPoints = modifiedPolygonCoordinates[0];
+    assert.lengthOf(modifiedPoints, 4);
+
+    const modifiedPoint = modifiedPoints[0];
+    assert.lengthOf(modifiedPoint, 2);
+    assert.notSameOrderedMembers(modifiedPoint, gent);
+
+    const unmodifiedPoint2 = modifiedPoints[1];
+    assert.lengthOf(unmodifiedPoint2, 2);
+    assert.sameOrderedMembers(unmodifiedPoint2, mechelen);
+
+    const unmodifiedPoint3 = modifiedPoints[2];
+    assert.lengthOf(unmodifiedPoint3, 2);
+    assert.sameOrderedMembers(unmodifiedPoint3, antwerpen);
+
+    const unmodifiedPoint4 = modifiedPoints[3];
+    assert.lengthOf(unmodifiedPoint4, 2);
+    assert.sameOrderedMembers(unmodifiedPoint4, leuven);
```

Comment:
kunt ge niet gewoon de geometry opvragen in GeoJson of zo, en dan een punt moven, en dan daarna een punt moven en checken dat de geometry niet meer hetzelfde is?

Reply:
idem opmerkingen testen hieronder

Reply:
heb de checks op coordinaten or uitgehaald, idem voor de andere testen. De de asserts nu zo herschreven zodat ze tegen het geometry object afchecken

Reply:
nog eens herschreven met gebruik van deepEquals